test/ttl_test.lua

```diff
@@ -9,6 +9,37 @@ g.before_all(function()
     g.queue_conn = config.cluster:server('queue-router').net_box
 end)
 
+local function lookup_task(task_id, tube_name, cluster)
+    local call_string = ("return box.space.%s:get(%s):tomap({names_only=true})"):format(tube_name, task_id)
```

Comment:
nit: `['%s']` is a bit more safe than `.%s.`

Reply:
it's local helper in test :)